src/models/MessageBody.ts

```diff
@@ -14,18 +14,27 @@
  * limitations under the License.
  ***************************************************************************** */
 
-export default interface MessageBody {
-	metadata: {
-		id: {
-			connectionId: {
-				sessionAlias: string;
+export default interface MessageBody extends Array<MessageBody> {
```

Comment:
let's make it similar to events body.
we will have MessageBodyPayload type
and `type MessageBody = MessageBodyPayload[]`;
then instead of passing index for rendering some payload you will pass this payload,